code/dashboard-2/src/components/Sidebar.tsx

```diff
@@ -22,16 +22,19 @@ import { LightDarkModeButton } from './LightDarkModeButton.tsx'
 const sidebarItems = [
   {
     path: '/dashboard/ml',
```

Comment:
On a general note, eventually this type of in-code configuration must disappear.